Tests/conf.json

```diff
@@ -992,7 +992,7 @@
         "Lastline": "Out of quota",
         "Netskope": "instance is down",
         "Joe Security": "Reached number of allowed submissions per month, untill 1/2/19",
-        "VirusTotal": "Reached number of allowed submissions"
+        "VirusTotal - Private API": "Reached number of allowed submissions"
```

Comment:
Please try to stick to the branch topic, skipping failing tests should be done from a different branch and then pull from master in this branch.

Reply:
So if I want to skip some tests, I need to change it only in master branch?

Reply:
If you want to skip a test, open a branch for it, skip it, merge to master, then pull from master in this branch.

Reply:
@adi88d please unskip the tests, and lets merge